### PR TITLE
test: [M3-7466] - Add integration test for Domains empty landing page

### PR DESCRIPTION
**Pull request by coliu-akamai**
## Description 📝
- integration test for domains empty landing page

### Verification steps 
- yarn cy:run -s "cypress/e2e/core/domains/domains-empty-landing-page.spec.ts"

## As an Author I have considered 🤔

*Check all that apply*

- [ ] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a changeset
- [x] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/cypress/e2e/core/domains/domains-empty-landing-page.spec.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import { ui } from 'support/ui';
```
Comment (coliu-akamai) [inline packages/manager/cypress/e2e/core/domains/domains-empty-landing-page.spec.ts:1]: saw the other domain test files start with 'smoke' -- should that be the case here as well? what does the 'smoke' part stand for?

Comment (jdamore-linode) [reply packages/manager/cypress/e2e/core/domains/domains-empty-landing-page.spec.ts:1]: No, this is fine!

The files prefixed with `smoke-` are meant to signify that they're smoke tests, or tests that confirm some crucial functionality without being too in-depth in its coverage, but we don't really follow the convention anymore (the `smoke-linode-landing-table.spec.ts` file has a lot of nitty gritty tests for the Linodes landing page that wouldn't really belong in a smoke test, for example).

We're long past the point of needing to rethink our test naming conventions and reorganize our test suite 😅

**Review by abailly-akamai — APPROVED** [review]
Nice test!

All passing locally ✅ and unit looks solid

File: packages/manager/cypress/e2e/core/domains/domains-empty-landing-page.spec.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    // confirms clicking on 'Create Domain' button
+    ui.button
+      .findByTitle('Create Domain')
+      .should('be.visible')
+      .should('be.enabled')
+      .click();
+
+    cy.url().should('endWith', '/domains/create');
+  });
+});
```
Comment (tyler-akamai) [inline packages/manager/cypress/e2e/core/domains/domains-empty-landing-page.spec.ts:74]: LGTM!

**Comment by tyler-akamai** [comment]
All tests pass

Screenshot 2023-12-19 at 11 01 02 AM

**Review by tyler-akamai — APPROVED** [review]

**Review by jdamore-linode — APPROVED** [review]
Great work @coliu-akamai!
